rsts/deployment/sandbox.rst

```diff
@@ -189,6 +189,6 @@ Flyte configuration on your remote cluster.
 
 #. You can now port-forward (or if you have load-balancer enabled then get an LB) to connect to remote FlyteConsole, as follows::
 
-    kubectl port-forward svc/envoy 30081:80
+    kubectl port-forward --address 0.0.0.0 svc/envoy 30081:80 --namespace projectcontour
```

Comment:
are we installing contour in the projectcontour namespace now?
cc @evalsocket

Reply:
yes, I updated some comments above.

Reply:
Sorry, I found that I accidentally create a branch in Flyte fork, I should create a branch in my fork. I am going to close it.

Reply:
sorry, I created a new PR. https://github.com/flyteorg/flyte/pull/1334